src/test/java/gumtree/spoon/AstComparatorTest.java

```diff
@@ -1541,8 +1527,9 @@ public void test_issue201812() throws Exception {
 		List<Operation> actions = result.getRootOperations();
 		result.debugInformation();
 
-		assertEquals(1, actions.size());
+		assertEquals(2, actions.size());
 		assertTrue(result.containsOperation(OperationKind.Insert, "Parameter", "call"));
+        assertTrue(result.containsOperation(OperationKind.Update, "TypeReference", "Callback<T>"));
```

Comment:
the type of this node should be "SUPER_TYPE" and not "TypeReference". could you have a look?

Reply:
Just to confirm, the above comment means that we need to create a new node type called SuperType which will be a subclass of TypeReference?

Are all the node types in this repo a subset of node types in Spoon?
If not, can you point me to implementation of such a Node?
If yes, I may need some help in figuring this out.

Reply:
> we need to create a new node type called SuperType 

Yes, a new Gumtree node type called `SUPER_TYPE`

> which will be a subclass of TypeReference?

No. In Gumtree nodes, there is no "subclass" relation ship

> Are all the node types in this repo a subset of node types in Spoon?

No. Some Gumtree nodes have no corresponding Spoon node types (eg `Modifiers`). `SUPER_TYPE` will be one of them.